docker/containers/docker_dev/Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG SOURCE=ghcr.io/neuralmagic/cuda-python3.8
+# how were these versions selected before?
+ARG TORCH_VERSION=1.10.2
+ARG TORCHVISION_VERSION=0.11.3
+ARG CUDA=113
+ARG BRANCH
+
+FROM $SOURCE
+
+ARG BRANCH
+
+RUN python3.8 -m pip install --upgrade pip \
+    && python3.8 -m pip install --upgrade setuptools
+
+ARG CUDA
+ARG TORCH_VERSION
+ARG TORCHVISION_VERSION
+
+RUN python3.8 -m pip install torch==${TORCH_VERSION}+cu${CUDA} torchvision==${TORCHVISION_VERSION}+cu${CUDA} -f https://download.pytorch.org/whl/torch_stable.html \
+    && git clone https://github.com/neuralmagic/sparseml.git --depth 1 --single-branch -b ${BRANCH} \
```

Comment:
If i make edits to the code, can i push the change using http git clone here?

Reply:
wdym?

Reply:
Sorry for the late response. The question is for pushing code if that is intended - ssh need ssh keys, http needs username + password /token. 
SparseML is public, so i dont think ssh/http will be an issue actually.